brilift/src/rt.rs

```diff
@@ -3,12 +3,12 @@ use std::mem::size_of;
 
 #[no_mangle]
 pub extern "C" fn print_int(i: i64) {
-    print!("{}", i);
+    print!("{i}");
```

Comment:
TIL!!!! Apparently this became possible in Rust 1.58 last year:
https://blog.rust-lang.org/2022/01/13/Rust-1.58.0.html#captured-identifiers-in-format-strings